lib/CMakeLists.txt

```diff
@@ -65,24 +65,13 @@ set_target_properties(vfio-user-static PROPERTIES
                       OUTPUT_NAME vfio-user
                       CLEAN_DIRECT_OUTPUT 1)
 
-set(UT_CFLAGS "-O0 -ggdb --coverage")
-set(UT_LFLAGS "--coverage")
-
-function(add_library_ut lib)
-    add_library(${lib} OBJECT ${ARGN})
-    set(lib_ut ${lib}_ut)
-    add_library(${lib_ut} ${ARGN})
-    set_target_properties(${lib_ut} PROPERTIES COMPILE_FLAGS ${UT_CFLAGS})
-    set_target_properties(${lib_ut} PROPERTIES LINK_FLAGS    ${UT_LFLAGS})
-endfunction(add_library_ut)
-
-add_library_ut(pci_caps pci_caps.c)
-add_library_ut(dma dma.c)
-add_library_ut(irq irq.c)
-add_library_ut(libvfio-user libvfio-user.c)
-add_library_ut(migration migration.c)
-add_library_ut(pci pci.c)
-add_library_ut(tran_sock tran_sock.c)
+add_library(pci_caps OBJECT pci_caps.c)
```

Comment:
Can all of these add_library() moved closer to set(LIBOBJS, ) at line:40 ?
This way if someone adds new file then its clear to also update set(LIBOBJS)